docs/UserGuide.adoc

```diff
@@ -411,12 +411,12 @@ _Constraints_ +
 
 // tag::find[]
 ==== Finding tasks by description or deadline: `find`
-Want to find a task? The find feature allows you to find a task using descriptions or deadline.
+Want to find a task? The find feature allows you to find task(s) using descriptions or deadline.
```

Comment:
task(s) by descriptions or deadlines*
My bad, I missed this in the initial review.